handler_test.go

```diff
@@ -97,39 +100,80 @@ func TestHandler(t *testing.T) {
 			goSource: "example.com/portmidi https://github.com/rakyll/portmidi _ _",
 		},
+		{
+			name:     "gotest.tools",
+			config:   gotestToolsConfig,
+			path:     "/",
+			goImport: "gotest.tools git https://github.com/gotestyourself/gotest.tools",
+			goSource: "gotest.tools https://github.com/gotestyourself/gotest.tools https://github.com/gotestyourself/gotest.tools/tree/master{/dir} https://github.com/gotestyourself/gotest.tools/blob/master{/dir}/{file}#L{line}",
+			redirect: "https://pkg.go.dev/gotest.tools/v3/",
+		},
+		{
+			name:     "gotest.tools/gotestsum",
+			config:   gotestToolsConfig,
+			path:     "/gotestsum",
+			goImport: "gotest.tools/gotestsum git https://github.com/gotestyourself/gotestsum",
+			goSource: "gotest.tools/gotestsum https://github.com/gotestyourself/gotestsum https://github.com/gotestyourself/gotestsum/tree/master{/dir} https://github.com/gotestyourself/gotestsum/blob/master{/dir}/{file}#L{line}",
+			redirect: "https://pkg.go.dev/gotest.tools/gotestsum/",
+		},
+		{
+			name:     "gotest.tools/assert",
+			config:   gotestToolsConfig,
+			path:     "/assert",
+			goImport: "gotest.tools git https://github.com/gotestyourself/gotest.tools",
+			goSource: "gotest.tools https://github.com/gotestyourself/gotest.tools https://github.com/gotestyourself/gotest.tools/tree/master{/dir} https://github.com/gotestyourself/gotest.tools/blob/master{/dir}/{file}#L{line}",
+			redirect: "https://pkg.go.dev/gotest.tools/v3/assert",
+		},
+		{
+			name:     "gotest.tools/v5/assert",
+			config:   gotestToolsConfig,
+			path:     "/v5/assert",
+			goImport: "gotest.tools git https://github.com/gotestyourself/gotest.tools",
+			goSource: "gotest.tools https://github.com/gotestyourself/gotest.tools https://github.com/gotestyourself/gotest.tools/tree/master{/dir} https://github.com/gotestyourself/gotest.tools/blob/master{/dir}/{file}#L{line}",
+			redirect: "https://pkg.go.dev/gotest.tools/v5/assert",
+		},
 	}
-	for _, test := range tests {
-		h, err := newHandler([]byte(test.config))
-		if err != nil {
-			t.Errorf("%s: newHandler: %v", test.name, err)
-			continue
-		}
-		s := httptest.NewServer(h)
-		resp, err := http.Get(s.URL + test.path)
-		if err != nil {
-			s.Close()
-			t.Errorf("%s: http.Get: %v", test.name, err)
-			continue
-		}
-		data, err := ioutil.ReadAll(resp.Body)
-		resp.Body.Close()
-		s.Close()
-		if resp.StatusCode != http.StatusOK {
-			t.Errorf("%s: status code = %s; want 200 OK", test.name, resp.Status)
-		}
-		if err != nil {
-			t.Errorf("%s: ioutil.ReadAll: %v", test.name, err)
-			continue
-		}
-		if got := findMeta(data, "go-import"); got != test.goImport {
-			t.Errorf("%s: meta go-import = %q; want %q", test.name, got, test.goImport)
-		}
-		if got := findMeta(data, "go-source"); got != test.goSource {
-			t.Errorf("%s: meta go-source = %q; want %q", test.name, got, test.goSource)
-		}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
```suggestion
		tc := tc
		t.Run(tc.name, func(t *testing.T) {
```

Reply:
Capturing the loop variable is only necessary if `t.Parallel()` is used, right?